packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,126 @@
+| `week-numbers`             | boolean                                                 | -                 | If true, show week numbers (1-53) before each week row                                                                            |
+| `min-weeks`                | number                                                  | -                 | Minimum number of weeks shown in a month                                                                                          |
+| `filter`                   | "week" \| "work-week" \| "4" \| "3" \| "2" \| "1"       | -                 | Allow user to view selected date with a range filter                                                                              |
+| `work-week`                | `[0, 1, 2, 3, 4, 5, 6]`                                 | `[1, 2, 3, 4, 5]` | The days that are selectable with the filter `work-week`. If the filter is not set to `work-week`, this does nothing.             |
+| `first-day-of-week`        | `0, 1, 2, 3, 4, 5, 6`                                   | `0`               | The first day of the week for locale                                                                                              |
+| `first-week-of-year`       | `0, 1, 2`                                               | `0`               | Enum that defines when the first week of the yera should start: first day (`0`), first full week (`1`), first four day week (`2`) |
+| `min-date`                 | string                                                  | -                 | The minimum allowable date                                                                                                        |
+| `max-date`                 | string                                                  | -                 | The maximum allowable date                                                                                                        |
+| `show-link`                | boolean                                                 | `true`            | If false, link at link slot is hidden                                                                                             |
+| `highlight-current-month`  | boolean                                                 | -                 | if true, the month picker should highlight the current month                                                                      |
+| `highlight-selected-month` | boolean                                                 | -                 | If true, the month picker should highlight the selected month                                                                     |
+| `today`                    | string                                                  | -                 | value of today                                                                                                                    |
```

Comment:
is the idea that the value would appear in the DOM as an attribute? or that you pass in the value of today? It's not totally clear.

Reply:
That's a great point. After looking at the FAST calendar element that I plan to extend from, I realized that they keep track of the value of today not as an attribute but it's declared in the calendar template itself. I put down "today" as an attribute based on the React Fluent calendar/date picker, but since the FAST calendar handles it differently, I can get rid of the attribute in my spec. However, that would be another difference between Fluent React and web components.

Reply:
> is the idea that the value would appear in the DOM as an attribute? or that you pass in the value of today? It's not totally clear.

That's a great point. After looking at the FAST calendar element that I plan to extend from, I realized that they keep track of the value of today not as an attribute but it's declared in the calendar template itself. I put down "today" as an attribute based on the React Fluent calendar/date picker, but since the FAST calendar handles it differently, I can get rid of the attribute in my spec. However, that would be another difference between Fluent React and web components.

Reply:
You raise a good point. Michelle. There WILL be instances where our WCs diverge from the Fluent React pattern. It's important to document those instances.